python/TestHarness/testers/Tester.py

```diff
@@ -80,6 +80,11 @@ def checkRunnable(self, options):
   def prepare(self):
     return
 
+  def getThreads(self, options):
+    return 1
+
+  def getProcs(self, options):
+    return 1
```

Comment:
Turns out that this isn't right for the JacobianAnalyzer. That should probably be fixed.